src/components/Buttons/docs/Button.md

```diff
@@ -0,0 +1,180 @@
+This is our Button.
+There are many like it, but this one is ours.
+
+Buttons guide our users to take a certain action, like 'Add to Cart' or 'Checkout'.
+The call to action should start with an active verb, and the style choice of the button should reflect the hierarchy and importance of the action.
+
+### Primary
+Primary buttons are for, you guessed it, the primary action.
+Don't use these lightly.
+Overuse on a single screen could hinder your main objective rather than help it.
+
+```js
+<div style={{display: 'flex'}}>
+  <div style={{marginRight: '24px'}}>
+    <div style={{marginBottom: '8px'}}>Tiny</div>
+    <Button snacksStyle="primary" size="tiny">
+      Primary Button
+    </Button>
+  </div>
+  <div style={{marginRight: '24px'}}>
+    <div style={{marginBottom: '8px'}}>Small</div>
+    <Button snacksStyle="primary" size="small">
+      Primary Button
+    </Button>
+  </div>
+  <div style={{marginRight: '24px'}}>
+    <div style={{marginBottom: '8px'}}>Standard</div>
+    <Button snacksStyle="primary" size="standard">
+      Primary Button
+    </Button>
+  </div>
+  <div>
+    <div style={{marginBottom: '8px'}}>Large</div>
+    <Button snacksStyle="primary" size="large">
+      Primary Button
+    </Button>
+  </div>
+</div>
+```
+
+### Secondary
+Secondary buttons are for your second most important action.
+A good use is as a cancel button when asking the user a question.
+
+```js
+<div style={{display: 'flex'}}>
+  <div style={{marginRight: '24px'}}>
+    <div style={{marginBottom: '8px'}}>Tiny</div>
+    <Button snacksStyle="secondary" size="tiny">
+      Secondary Button
+    </Button>
+  </div>
+  <div style={{marginRight: '24px'}}>
+    <div style={{marginBottom: '8px'}}>Small</div>
+    <Button snacksStyle="secondary" size="small">
+      Secondary Button
+    </Button>
+  </div>
+  <div style={{marginRight: '24px'}}>
+    <div style={{marginBottom: '8px'}}>Standard</div>
+    <Button snacksStyle="secondary" size="standard">
+      Secondary Button
+    </Button>
+  </div>
+  <div>
+    <div style={{marginBottom: '8px'}}>Large</div>
+    <Button snacksStyle="secondary" size="large">
+      Secondary Button
+    </Button>
+  </div>
+</div>
+```
+
+### Coupon
+
+Note that coupon buttons aren't affected by a Snacks theme.
```

Comment:
Did we confirm with retailer team that this is what we want?

Reply:
@dcocchia checking now, I'm not sure which theme color we'd want to use here. We'd likely need to add an option, even the default IC theme wouldn't have something to cover coupons.

Reply:
confirmed that coupons aren't themable